index.js

```diff
@@ -37,28 +37,29 @@ async function addRouteThrottleHook (fastify, routeOptions, throttleOptions) {
 
 function throttleOnSendHandler (fastify, throttleOpts) {
   const bytesPerSecond = throttleOpts.bytesPerSecond
+  const pipelineErrorCallback = err => { if (err) fastify.log.error(err) }
```

Comment:
I would prefer to call it pipelineCallback, because it is not only called in the error case. 
Also you can simplify the function to this:

```suggestion
  const pipelineCallback = err => err && fastify.log.error(err)
```

Other than that LGTM

Reply:
Ok, I applied the suggested changes